templatestore/frontend/src/components/Dropdowncomp.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import styles from '../style/templateScreen.less';
+export default function Dropdowncomp(props) {
+    return (
+        <div className={styles.teAttributesRow}>
+            <div className={styles.teAttributesCell}>
+                <select
+                    name="urlKey"
+                    value={props.urlKey}
+                    placeholder="Choose a value"
+                    onChange={(e) => props.handleChange(e, props.id)}
+                >
+                    <option value="" disabled>Choose URL</option>
+                    {props.listOfUrls.map((urlopt) => (
+                        <option value={urlopt} disabled={props.urlKey !== urlopt && urlopt in props.visited} >{urlopt}</option>
+                    ))}
+                </select>
+            </div>
+            <div className={styles.teAttributesCell}>
+                <select
+                    name="expiry"
+                    value={props.expiry}
+                    placeholder="Choose a value"
+                    onChange={(e) => props.handleChange(e, props.id)}
+                >
+                    <option value="" disabled>Choose Expiry</option>
+                    <option value="1,0,0">1 Day</option>
```

Comment:
Can we just have the number of days here? These easily get confusing to remember and parse. Also, do you think it will be better to have all the options in days? Currently, it's slightly unclear what does a month means? 30 days? 31 days? 6 months in little more unclear.

Reply:
I relied on js Date method for this, i think that handles the 30 and 31 month thing.